examples/fingerprint_simpletest.py

```diff
@@ -1,17 +1,19 @@
 import time
 import board
-import busio
+#import busio
 from digitalio import DigitalInOut, Direction
 import adafruit_fingerprint
+import serial
 
 led = DigitalInOut(board.D13)
 led.direction = Direction.OUTPUT
 
-uart = busio.UART(board.TX, board.RX, baudrate=57600)
+#uart = busio.UART(board.TX, board.RX, baudrate=57600)
```

Comment:
don't change the default example setup

Reply:
reverted the default example and created a new one.